tools/cloud_functions/gcs_event_based_ingest/tests/conftest.py

```diff
@@ -134,6 +134,30 @@ def teardown():
     return data_objs[-1]
 
 
+@pytest.fixture(scope="function")
+@pytest.mark.usefixtures("gcs_bucket", "dest_dataset", "dest_table")
+def gcs_data_under_sub_dirs(request, gcs_bucket, dest_dataset,
```

Comment:
Can you add docstring with brief explanation of test. The test name isn't clear as to what it's testing.